Pages/Table.php

```diff
@@ -989,9 +1063,12 @@ public function render_action_header() {
                 echo "<a href='".$this->createUrl('new') . "'><img src='/images/lightning/new.png' border='0' title='Add New' /></a>";
             }
             if ($this->importable) {
-                echo "<a href='".$this->createUrl('import') . "'><img src='/images/lightning/send_doc.png' border='0' title='Import' /></a><br />";
+                echo "<a href='".$this->createUrl('import') . "'><img src='/images/lightning/send_doc.png' border='0' title='Import' /></a>";
+            }
+            if ($this->exportable) {
+                echo "<a  onclick='lightning.table.export(\"{$this->createUrl('export')}\")'><img src='/images/lightning/detach.png' border='0' title='Export' /></a><br />";
```

Comment:
i think the url is already saved in the js, so it's not necessary to pass it to this function